src/app/file-browser/config/file-browser-split.config.ts

```diff
@@ -17,7 +17,9 @@ export class FileBrowserSplitConfig extends TerraMultiSplitViewConfig
     private _imagePreviewView:TerraMultiSplitViewInterface;
     private _storageServices:Array<TerraBaseStorageService>;
 
-    public init(storageServices:Array<TerraBaseStorageService>):void
+    public isImagePreviewEnabled:boolean;
```

Comment:
Wird die Variable überhaupt verwendet? In der Abfrage verwendest du `storageService.isImagePreviewEnabled`